docs/contact/index.html

```diff
@@ -279,30 +279,24 @@ <h1 class="text-3xl-5xl font-normal">Follow HF Developments</h1>
   <div class="mt-10 md:mt-12 lg:mt-16 xl:mt-20 ">
     <div class="grid gap-8 md:grid-cols-2">
   <div class="bg-white border border-gray-300 rounded py-8 px-6 sm:px-12 text-center">
-    <h3 class="font-semibold text-xl">Haskell Foundation Discourse</h3>
+    <h3 class="font-semibold text-xl">HF Discourse</h3>
     <p class="mt-6">
-      This list will have infrequent traffic announcing major news from the Haskell Foundation. It is expected that all messages will come from HF or its designees.
+      Discord is where HF conducts presents its major announcements, and solicits feedback on proposals and ongoing development. Additionally, this is a great place to make your desire to participate known, and to get up to date meeting minutes.
```

Comment:
The official home of the meeting minutes is at https://gitlab.haskell.org/hf/minutes. Is that worth linking somewhere (perhaps not here)?

Reply:
I think it's worth adding to the discourse card. I'll have that up in the next MR